relay-cardinality/src/redis/cache.rs

```diff
@@ -1,11 +1,17 @@
 use std::fmt;
 use std::sync::{PoisonError, RwLock, RwLockReadGuard, RwLockWriteGuard};
+use std::time::Duration;
 
 use relay_common::time::UnixTimestamp;
+use relay_statsd::metric;
 
 use crate::redis::QuotaScoping;
+use crate::statsd::{CardinalityLimiterCounters, CardinalityLimiterTimers};
 use crate::window::Slot;
 
+/// Interval in which the cache is checked for expired values.
+const CACHE_VACUUM_INTERVAL: Duration = Duration::from_secs(180);
```

Comment:
Should we make this configurable right away? Seems like something that belongs in the static config.